### PR TITLE
Feature/my questionnaire view

### PR DESCRIPTION
**Pull request by ghost**
A bit tricky with the view, any suggestions to make more space between the questions?

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4,6 +4,18 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+<<<<<<< HEAD
```
Comment (SebastianGerS) [inline package-lock.json:7]: you have some un resolved merge conflict beacous this is a package-lock.json file you might be better of deleting this file and then running npm install

File: src/Views/Site/MyQuestion.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+
```
Comment (SebastianGerS) [inline src/Views/Site/MyQuestion.js]: this is my fault i guess but I was just trying to ressolve the fact that you had not created a view, so we move your component to the view folder, quick and easy, but realy what we should have done is to create a new view and used the just imported this component into that view.

so what you can do here is just move all the code in this document to a new component that you then import into this view (you can look att the code in my CreateQuestion view to get the right structure altought you shulde make it a statless component (which I should correct in mine). just remember to hcange the path to the pole component.

File: src/Views/Site/MyQuestion.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    const { activeIndex } = this.state
+    const newIndex = activeIndex === index ? -1 : index
+
+    this.setState({ activeIndex: newIndex })
+  }
+
+  render() {
+    const { activeIndex } = this.state
+
+    return (
```
Comment (SebastianGerS) [inline src/Views/Site/MyQuestion.js]:
```
return (
     
            <Accordion  fluid className="center-content-grid">
              <div className="margin-tb-2 padding-1">
                <Accordion.Title  className="space-between frame minwidth-10" active={activeIndex === 0} index={0} onClick={this.handleAccordion}>
                    Do you like dogs?
                    <Icon name='dropdown' className="minwidth-10" />
                </Accordion.Title>
                  <Accordion.Content  active={activeIndex === 0}>
                  <Poll />
                  </Accordion.Content>
              </div>
```

.... osv

File: src/Components/Poll.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+    this.togglePoll = this.togglePoll.bind(this)
+  }
+
+  togglePoll() {
+    this.setState({ active: !this.state.active })
+  }
+
+  render() {
+    return(
```
Comment (SebastianGerS) [inline src/Components/Poll.js]:
```
return(
        <div  className ="center-content-column padding-1">
            <div className="center content-row">
            <button class="ui blue basic button"  >{this.state.active ? "Live results" : "Edit" }</button>
            <button class="ui orange basic button"  onClick={this.togglePoll}>{ this.state.active ? "End" : "Activate" }</button>
            </div>
            { this.state.active &&
            <div class="center-content-column padding-1">
                <h3>Share Poll</h3>
                <div className="center-content-row">
                <button class="ui basic button yellow" >Link</button>
                <button class="ui basic button yellow"  >QR-Code</button>
                </div>
            </div>
            }
          </div>  
        
    );
```

File: src/Assets/Styles/Views/Site/MyQuestion.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+}
+.center-content{
+    display:flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-content: space-between;
+    align-items: center;
+}
+
```
Comment (SebastianGerS) [inline src/Assets/Styles/Views/Site/MyQuestion.scss]: la till följande klasser

```
.center-content-column {
    display:flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
}


.frame {
    border: 1px solid #FE9A76;
}

.margin-tb-2 {
    margin: 2em 0;
} 

.minwidth-10 {
    min-width: 10em;
}

.space-between  {
    display: flex;
    flex-direction: row;
    align-items: space-between;
}

.padding-1 {
    padding: 1em;
}
```

**Review by SebastianGerS — COMMENTED** [review]
the code I inserted solves some of the problems with the view, but not all of them, but you may use my suggestions as a starting point of fixing the rest of the problems

**Comment by ghost** [comment]
Thank you for excellent help!

2018-04-21 0:11 GMT+02:00 SebastianGerS <notifications@github.com>:

**Pull request merged**